packages/clerk-js/src/core/fapiClient.test.ts

```diff
@@ -116,6 +162,23 @@ describe('request', () => {
     );
   });
 
+  it('invokes global.fetch with proxy', async () => {
+    // window.location.href = 'http://clerk.example.com';
```

Comment:
❓Commenting this like did not affect the results of the tests, was that necessary ?

Reply:
You can remove it then.